README.md

```diff
@@ -12,12 +12,12 @@ Below is a sample command line that shows how the plot clip image could be run.
 An explanation of the command line options used follows.
 Be sure to read up on the [docker run](https://docs.docker.com/engine/reference/run/) command line for more information.
 
-The files that are used in this example are available through Google Drive: [ua_gantry_plot_clip_test_data.tar.gz](https://drive.google.com/file/d/17b7328H6B3olwesqKyjxYjfEDJiQz6N_/view?usp=sharing)
+The files that are used in this example are available through Google Drive: [plotclip_sample_data.tar.gz](https://drive.google.com/file/d/1AHx6surpXV7izII2hn3c2wn_qnj5dvCb/view?usp=sharing).
```

Comment:
For future, I think having the file somewhere (like a public link on Data Store) which a user can download using wget/curl would be preferable than having to manually download and unzip.

Reply:
I'm going to change this for PR https://github.com/AgPipeline/transformer-plotclip/pull/20